src/Microsoft.Health.Fhir.SqlServer/Features/Operations/Import/SqlImportOperation.cs

```diff
@@ -85,11 +85,7 @@ public IReadOnlyList<(Table table, Index index)> UniqueIndexesList()
         {
             var list = new List<(Table table, Index index)>();
 
-            if (_schemaInformation.Current >= SchemaVersionConstants.RenamedIndexForResourceTable)
-            {
-                list.Add((VLatest.Resource, VLatest.Resource.IX_Resource_ResourceSurrogateId));
-            }
```

Comment:
May still be true between versions `SchemaVersionConstants.RenamedIndexForResourceTable` and <34

Reply:
It is overkill to support all permutations for functionality that nobody is using... I suggest let it go.

Reply:
Added different skip. This will result in this index not being disabled, when it exists. Code will still work, and on final version it will perform.